src/SFA.DAS.Approvals/InnerApi/Requests/PostAddAndApproveDraftApprenticeshipsRequest.cs

```diff
@@ -0,0 +1,18 @@
+﻿using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.Approvals.InnerApi.Requests
+{
+    public class PostAddAndApproveDraftApprenticeshipsRequest : IPostApiRequest
+    {
+        public string PostUrl => $"api/{ProviderId}/bulkupload/AddAndApprove";
+
+        public PostAddAndApproveDraftApprenticeshipsRequest(long providerId, BulkUploadAddAndApproveDraftApprenticeshipsRequest data)
+        {
+            ProviderId = providerId;
+            Data = data;
+        }
+
+        public long ProviderId { get; }
```

Comment:
this property is unused, also could we consistently place all properties above the constructor?